src/commands/Misc/Birthday/upcoming-birthdays.ts

```diff
@@ -0,0 +1,46 @@
+import { SkyraCommand } from '#lib/structures';
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Message, MessageEmbed } from 'discord.js';
+import { DbSet, ScheduleEntity } from '#lib/database';
+
+@ApplyOptions<SkyraCommand.Options>({
+	cooldown: 5,
+	description: LanguageKeys.Commands.Misc.UpcomingBirthdaysDescription,
+	extendedHelp: LanguageKeys.Commands.Misc.UpcomingBirthdaysExtended,
+	runIn: ['text']
+})
+export default class extends SkyraCommand {
+	public async run(message: Message, args: SkyraCommand.Args) {
+		const schedules = [
+			...this.context.client.schedules.queue
+				.filter((schedule) => schedule.taskID === 'birthday' && schedule.data.guildID === message.guild!.id)
+				.reduce((a, b) => {
+					const key = b.time.getTime();
+					if (a.has(key)) a.get(key)!.push(b);
+					else a.set(key, [b]);
+					return a;
```

Comment:
```suggestion
					const valueAtA = a.get(key);
					valueAtA ? valueAtA.push(b) : a.set(key, [b]);
					return a;
```

This way the map doesn't need to be accessed twice.